modules/button/css/lib/icon-button.scss

```diff
@@ -60,7 +61,7 @@
 
       .wd-icon-accent,
       .wd-icon-fill {
-        fill: $wdc-btn-icon-disabled-color;
+        fill: $wdc-btn-icon-disabled-color !important;
```

Comment:
Because of JS injector for icons, we need to add !important to override those styles that are coloring the icons.